frontend/src/components/sign/components/sign-with-oauth/sign-with-oauth.tsx

```diff
@@ -33,7 +33,7 @@ const SignWithGoogle = ({ title }: { title: string }): React.ReactElement => {
     };
   }, [onStorageEvent]);
 
-  return <ButtonOutline text={title + ' with Google'} onClick={onGoogleSign} />;
+  return <ButtonOutline text={'Sign In with ' + title} onClick={onOAuthSign} />;
```

Comment:
It looks like we gonna display _Sign In_ on both `Sign In` and `Sign Up` pages. Maybe it will be better if we change it to `Continue with`